aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -141,21 +198,10 @@ module aptos_framework::genesis {
         let num_validators = vector::length(&validators);
         while (i < num_validators) {
             let validator = vector::borrow(&validators, i);
-            let owner = &account::create_account(validator.owner_address);
-            let operator = owner;
-            // Create the operator account if it's different from owner.
-            if (validator.operator_address != validator.owner_address) {
-                operator = &account::create_account(validator.operator_address);
-            };
-            // Create the voter account if it's different from owner and operator.
-            if (validator.voter_address != validator.owner_address &&
-                validator.voter_address != validator.operator_address) {
-                account::create_account(validator.voter_address);
-            };
-
-            // Mint the initial staking amount to the validator.
-            coin::register<AptosCoin>(owner);
-            aptos_coin::mint(aptos_framework, validator.owner_address, validator.stake_amount);
+
+            let owner = &create_account(aptos_framework, validator.owner_address, validator.stake_amount);
```

Comment:
I think this is another point, if the stake amount exists, but the account was missed in the previous list, there will be a mismatch and more coins will be made than expected.

Also, should assert that the stake amount is less than or equal to its current balance if we are pulling the new account at this point.

Reply:
I'll double check the staking code, but the ideal scenario would be if we try to stake more than we have abort or just stake what we have.... we can discuss what is ideal behavior.

Reply:
Yes, if we offer to stake less than what we have, it will error out. We do know that we likely do NOT want to stake everything, we will need to at least keep a few apt in the account for performing actions.

Reply:
Sounds good, just keeping in mind that we should check what we can.

Reply:
We actually might want to stake less that we have and keep some for gas? Otherwise the owner account cannot do anything else such as reset lockup or update operator/voter